bip-PSBT-SP.mediawiki

```diff
@@ -0,0 +1,262 @@
+<pre>
+  BIP: ?
```

Comment:
Assigned 375.

Please update here, add a README entry, and update the `Created` field to today's date.